staging/src/k8s.io/apiserver/pkg/authorization/path/path.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package path
```

Comment:
Add package comment

Reply:
in path.go? If we do that, then in doc.go. Adding.

Reply:
done

Reply:
I personally see no reason to add a separate file for that, but if that's the k8s policy, so be it.